data/report_templates/ont_project_summary.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Project Report
+subtitle: {{ project.ngi_name }}_project_summary
+date: {{ report_info.report_date }}
+support_email: {{ report_info.support_email }}
+---
+
+# Project Information
+
+User Project Name
+:   {{ project.user_ID }}
+
+NGI Project Name
+:   {{ project.ngi_name }}
+
+NGI Project ID
+:   {{ project.ngi_id }}
+
+{% if project.ngi_facility -%}
+NGI Facility
+:   {{ project.ngi_facility }}
+{%- endif %}
+
+User Contact
+:   [{{ project.contact }}](mailto:{{ project.contact }})
+
+NGI Application Type
+:   {{ project.application }} _({% if project.best_practice %}including best practice analysis{% else %}no best practice analysis{% endif %})_
+
+{% if project.num_samples -%}
+Submitted samples
+:   {{ project.num_samples }} sample{% if project.num_samples > 1 %}s{% endif %}
+{%- endif %}
+
+{% if project.num_lanes -%}
+Ordered Flow cells
+:   {{ project.num_lanes }} flow cell{% if project.num_lanes > 1 %}s{% endif %}
+{%- endif %}
+
+Order Dates
+:   {{ report_info.dates }}
+
+{% if project.reference.genome -%}
+Reference Genome
+:   {% if project.reference.organism -%}{{ project.reference.organism}} - {% endif %}{{ project.reference.genome }}
+{%- endif %}
+
+Report generated by
+:   {{ report_info.signature }}, {{ report_info.report_date }}
+
+NGI is an accredited facility. However, the workflows used for this project is not under accreditation.
+
+# Methods
+
+{% if project.library_construction -%}
+### Library construction
+{{ project.library_construction }}
+{%- endif %}
+
+{% if report_info.sequencing_methods -%}
+### Sequencing
+{{ report_info.sequencing_methods }}
+{%- endif %}
+
+### Data Processing
+To ensure that all sequenced data meets our guarantee of data quality and quantity,
+a number of standardised bioinformatics quality control checks are performed before
+delivery. These include checking the yield, sequence read quality and average read length.
+
+
+# Sample Information
+{% if not project.samples %}
+No sample information to be displayed.
+{% else %}
+NGI ID | User ID | {{ project.samples_unit }}
+-------|---------|----------
+{% for sample in project.samples.values()|sort(attribute='ngi_id') -%}
+{{ sample.ngi_id }} | `{{ sample.customer_name }}` | {{ sample.total_reads }}
+{% endfor %}
+
+
+Below you can find an explanation of the header column used in the table.
+
+{{ tables.sample_info }}
+{% endif %}
+
+
+# Library Information
+{% if not project.samples %}
+No library information to be displayed.
+{% else %}
+NGI ID | Index | Avg. FS (bp) | Lib. QC
+-------|-------|--------------|--------
+{% for sample in project.samples.values()|sort(attribute='ngi_id') -%}
+{% if sample.preps -%}
+{% for prep in sample.preps.values() -%}
+{{ sample.ngi_id }} | `{{ prep.barcode }}` | {{ prep.avg_size }} | {{ prep.qc_status }}
+{% endfor -%}
+{% endif -%}
+{%- endfor %}
+
+
+Below you can find an explanation of the header column used in the table.
+
+{{ tables.library_info }}
+{% endif %}
+
+
+# Flow cell Information
```

Comment:
If a lot of the info is the same could we not split the md file into two, and include one in the other like so? 

`{% include 'fc_specific_template.md' %}`

Reply:
Hm, sounds interesting. Some fields are the same, like the first and last ones that have more general info. But the tables are quite different. I guess we'd need 3 files in that case, one general and one for each application.

Reply:
Done!